kindlecomicconverter/comic2ebook.py

```diff
@@ -1154,9 +1163,17 @@ def makeBook(source, qtgui=None):
     checkPre(source)
     print("Preparing source images...")
     path = getWorkFolder(source)
+    pathOebpsImages = os.path.join(path, "OEBPS", "Images")
     print("Checking images...")
-    getComicInfo(os.path.join(path, "OEBPS", "Images"), source)
-    detectCorruption(os.path.join(path, "OEBPS", "Images"), source)
+    getComicInfo(pathOebpsImages, source)
+    listFiles, listCbzFiles, countCbzFiles = listFilesS(pathOebpsImages), listCbzFilesS(pathOebpsImages), countCbzFilesS(pathOebpsImages)
+    if countCbzFiles:
+        for file in listCbzFiles:
+            KCC_gui.WorkerThread.displayProgressMessage(file)
+            makeBook(os.path.join(source, file)) 
```

Comment:
This fails if 7z is not installed. 7z/homebrew requirement was dropped in #693

Reply:
Good to know. Is it okay to update readme? I can do it.

Reply:
We'll update it when 6.1 releases to not confuse users.